lib/trento_web/controllers/v1/suma_credentials_controller.ex

```diff
@@ -24,4 +24,19 @@ defmodule TrentoWeb.V1.SUMACredentialsController do
       render(conn, "suma_credentials.json", %{settings: settings})
     end
   end
+
+  operation :delete,
+    summary: "Clears the SUMA credentials",
+    tags: ["Platform"],
+    description: "Clears the saved credentials for SUSE Manager",
+    responses: [
+      no_content: "Settings cleared successfully"
+    ]
+
+  @spec delete(Plug.Conn.t(), any) :: Plug.Conn.t()
+  def delete(conn, _) do
+    with :ok <- SoftwareUpdates.clear_settings() do
+      send_resp(conn, :no_content, "")
+    end
```

Comment:
```suggestion
   :ok = SoftwareUpdates.clear_settings()
   send_resp(conn, :no_content, "")
```

You can just match that result, no need for the `with` :wink: